jemalloc/src/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -ex
+
+CWD=$PWD
+
+cd /tmp
+
+rm -rf /tmp/usr
+mkdir -p /tmp/usr
+
+rm -rf jemalloc_latest
+mkdir -p jemalloc_latest
+
+cd jemalloc_latest
+
+git clone https://github.com/jemalloc/jemalloc.git
+cd jemalloc
+
+autoconf
+./configure --prefix=/tmp/usr --with-jemalloc-prefix=je
```

Comment:
I'd prefer a subfolder of current build folder being used instead of /tmp. Location of temporary folders is system dependent and not very portable.